posts/2022-10-27-liberty-tools-eclipse-jakarta-ls.adoc

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "Jakarta EE with Liberty Tools for Eclipse IDE"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/awisniew90
+author_github: https://github.com/awisniew90
+seo-title: Jakarta EE with Liberty Tools for Eclipse IDE
+seo-description: Liberty tools for Eclipse IDE now supports Jakarta EE application development 
+blog_description: "Liberty tools for Eclipse IDE now supports Jakarta EE application development "
+open-graph-image: https://openliberty.io/img/blog/liberty-tools-eclipse-jakarta-hover.png
+---
+= Jakarta EE with Liberty Tools for Eclipse IDE
+Adam Wisniewski <https://github.com/awisniew90>
+
+Since link:https://openliberty.io/blog/2022/08/01/liberty-tools-eclipse.html[introducing Liberty Tools for Eclipse IDE] in August, we've continued to add new features with each new early release. 
+Currently at link:https://github.com/OpenLiberty/liberty-tools-eclipse/releases/tag/liberty-tools-0.4.0[version 0.4], Liberty Tools now supports developing Jakarta EE applications and editing Liberty's `bootstrap.properties` and `server.env` files.
+
+Let's take a closer look.
+
+== Jakarta EE development
+
+Integration with the Eclipse Language Server for Jakarta EE (Eclipse LSP4Jakarta) allows users to intuitively develop with link:https://jakarta.ee/[Jakarta EE]. Features such as autocompletion, quick-fixes and type-ahead seamlessly integrate into your day-to-day development making it easy to find the right methods and annotations for your application.
+
+[.img_border_light]
+image::/img/blog/liberty-tools-eclipse-jakarta-quick-fix.gif[Jakarta Quick Fix,width=70%,float="center"]
+
+Additionally, full code snippets make development quick and easy.
```

Comment:
can we say a little more about this- from looking at the image, i think "full code snippets" means pre-configured annotations and methods. Just a little more detail (sentence or two) would be helpful I think